Routes/index.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const auth = require('../Controllers/auth');
+const user = require('../Controllers/user');
+const app = express();
+
+app.post('/login', auth.login);
```

Comment:
I guess in your code, routes are controllers and controllers in your code should be modals.

Reply:
No..controllers are controllers...they are handler functions right..i havent exclusively created models for actions on db..